utils.h

```diff
@@ -226,10 +226,31 @@ inline std::string size_attr_str(const Token& t, const std::string& prefix = "")
     return count_attr_str(t, prefix);
 }
 
+inline std::string pcount(Decl* p, const std::string& prefix = "")
+{
+    if (p->attrs_ && (p->attrs_->string_ || p->attrs_->wstring_))
+        return prefix + p->name_ + "_len";
+    if (p->dims_ && !p->dims_->empty())
+        return "(size_t)1";
+    if (p->type_->tag_ == Foreign && p->attrs_ && p->attrs_->isary_)
+        return "(size_t)1";
+
+    if (!p->attrs_->size_.is_empty() && !p->attrs_->count_.is_empty())
+        return size_attr_str(p->attrs_->size_, prefix);
```

Comment:
Why do we prefer size when both size and count are specified?

Reply:
Fixed.